app/services/notices/setup/check.service.js

```diff
@@ -9,6 +9,7 @@ const CheckPresenter = require('../../../presenters/notices/setup/check.presente
 const DetermineRecipientsService = require('./determine-recipients.service.js')
 const PaginatorPresenter = require('../../../presenters/paginator.presenter.js')
 const RecipientsService = require('./fetch-recipients.service.js')
+const FetchAbstractionAlertContactsService = require('./fetch-abstraction-alert-recipients.service.js')
```

Comment:
```suggestion
const FetchAbstractionAlertRecipientsService = require('./fetch-abstraction-alert-recipients.service.js')
```

I'd also suggest renaming the variable `RecipientsService` to `FetchRecipientsService`, then double-checking the order of the `require()` statements.